processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -130,3 +140,22 @@ func reverseLookupHost(d *Detector) (string, error) {
 	}
 	return hostname, nil
 }
+
+func setHostCPUInfo(d *Detector, cpuInfo cpu.InfoStat) {
+	d.rb.SetHostCPUVendorID(cpuInfo.VendorID)
+	family, err := strconv.ParseInt(cpuInfo.Family, 10, 64)
+	if err != nil {
+		d.logger.Debug("failed to get host cpuinfo family", zap.Error(err))
```

Comment:
nit: I think we should return the error here instead and log with a warning as part of `Detect`